### PR TITLE
fix(fast-form-generator): remove resize behavior and update fontfamily

### PR DESCRIPTION
**Pull request by eljefe223**
Closes: #726 
 
For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-form-generator-react/src/form/form-item.textarea.style.ts [hunk 1]
```diff
@@ -17,7 +17,9 @@ const styles: ComponentStyles<IFormItemTextareaClassNameContract, {}> = {
     formItemTextarea_textarea: {
         ...applyInputStyle(),
         marginTop: toPx(8),
-        marginBottom: toPx(2)
+        marginBottom: toPx(2),
+        resize: "none",
+        fontFamily: "Segoe UI, SegoeUI, Helvetica Neue, Helvetica, Arial, sans-serif"
```
Comment (chrisdholt) [inline packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: Does "inherit" not work here? If it doesn't, we should consider setting a variable somewhere and importing that. Having multiple instances of these strings throughout the package will prove difficult to maintain.

Comment (eljefe223) [reply packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: yes "inherit" does work.

File: packages/fast-form-generator-react/src/form/form-item.textarea.style.ts [hunk 2]
```diff
@@ -17,7 +17,9 @@ const styles: ComponentStyles<IFormItemTextareaClassNameContract, {}> = {
     formItemTextarea_textarea: {
         ...applyInputStyle(),
         marginTop: toPx(8),
-        marginBottom: toPx(2)
+        marginBottom: toPx(2),
+        resize: "none",
```
Comment (nicholasrice) [inline packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: Did you consider just restricting to vertical resize? I understand why we wouldn't want to do a horizontal resize as it doesn't align with the container but I could see how vertical resize could be nice if typing long paragraphs.

Comment (chrisdholt) [reply packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: I think part of the reason for removing here is that it has poor cross-browser support (no IE or Edge support at all). For a consistent experience we discussed simply removing this all up.

Comment (nicholasrice) [reply packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: My opinion is that we should keep vertical if its available - I don't think we should remove useful features just get consistency.

Comment (chrisdholt) [reply packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: I'm fine with this ^^^

Comment (eljefe223) [reply packages/fast-form-generator-react/src/form/form-item.textarea.style.ts]: Just verified that `resize: "vertical"` works in firefox and chrome

**Pull request merged**